core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -283,8 +283,10 @@ enum class UniqueType(
     CostIncreasesPerCity("Cost increases by [amount] per owned city", UniqueTarget.Building, UniqueTarget.Unit),
     CostIncreasesWhenBuilt("Cost increases by [amount] when built", UniqueTarget.Building, UniqueTarget.Unit),
 
+    @Deprecated("as of 4.10.17", ReplaceWith("Only available <if [buildingFilter] is constructed in all [non-[Puppeted]] cities>"))
     RequiresBuildingInAllCities("Requires a [buildingFilter] in all cities", UniqueTarget.Building),
-    RequiresBuildingInSomeCities("Requires a [buildingFilter] in at least [amount] cities", UniqueTarget.Building),
+    @Deprecated("as of 4.10.17", ReplaceWith("Only available <if [buildingFilter] is constructed in at least [positiveAmount] [All] cities>"))
```

Comment:
HMMM
Another bad one
We can start with replacing only the "all" unique for now, since that seems fine
Or we can find a solution for this as well 
I mean, we could go with "non-puppeted" for this as well, perhaps...? But that does change the meaning, and makes it more restrictive, rather than less...

Reply:
"of your"? "owned"? 
"of your" sounds like the best *English*, but I wonder if other languages will have difficulty..."owned" sounds like the correct solution for a language-foible-agnostic solution

Reply:
Well that's goes back to my comment [above](https://github.com/yairm210/Unciv/pull/11101#issuecomment-1937380146). "Of [cityFilter] cities" reads best for some filters, less so for others. Kinda split on it myself tbh